djqscsv/djqscsv.py

```diff
@@ -36,19 +44,33 @@ def render_to_csv_response(queryset, filename=None, append_datestamp=False,
         filename = generate_filename(queryset,
                                      append_datestamp=append_datestamp)
 
-    response = HttpResponse(content_type='text/csv')
+    if not kwargs.get('streaming', True):
+        response = HttpResponse(content_type='text/csv')
+        write_csv(queryset, response, **kwargs)
+    else:
+        response = StreamingHttpResponse(
+            _iter_csv(queryset, _Echo(), **kwargs),
+            content_type='text/csv')
```

Comment:
Consider pulling `content_type` out into a dict assigned to a variable and then unpacking it into `HttpResponse()` and `StreamingHttpResponse()` with `**`.